Clarified format of postgres_dsn (#7139)
mattermost/docs: source/deploy/postgres-migration.rst

.. code:: sh

>    dbcmp --source "${MYSQL_DSN}" --target "postgres://${POSTGRES_DSN} " --exclude="db_migrations,ir_,focalboard,systems"

Reviewer: ```suggestion
   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN} " --exclude="db_migrations,ir_,focalboard,systems"
```

> For example: ``dbcmp --source "user:password@tcp(address:3306)/db_name --target "postgres://user:password@address:5432/db_name``
> 
> .. note::
>    
>    This migration guide only covers the tables for Mattermost products.

Reviewer: ```suggestion
An example command would look like: ``dbcmp --source "user:password@tcp(address:3306)/db_name --target "postgres://user:password@address:5432/db_name``

.. note::
   
   ``POSTGRES_DSN`` should start with a ``postgres://`` prefix. This way ``dbcmp`` decides which driver to use while connecting to a database.
```

.. code:: sh

>    dbcmp --source "${MYSQL_DSN}" --target "postgres://${POSTGRES_DSN}" --exclude="db_migrations,systems"

Reviewer: ```suggestion
   dbcmp --source "${MYSQL_DSN}" --target "${POSTGRES_DSN}" --exclude="db_migrations,systems"
```
